portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,8 +46,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Would you mind adding a comment in a Javadoc style to explain what this function is doing? All public functions should have documentation.

http://www.drjava.org/docs/user/ch10.html

Reply:
Solved